src/cap-description.adoc

```diff
@@ -475,8 +475,9 @@ while still allowing the bounds to be correctly decoded.
 .Memory address bounds encoded within a capability
 image::cap-bounds-map.png[width=80%,align=center]
 
-A capability whose bounds cover the entire address space has 0 base and top
-equals 2^MXLEN^, i.e. _t_ is a MXLEN + 1 bit value. However, _b_ is a
+A capability has _infinite_ bounds if its bounds cover the entire address space
+such that the base address _b_=0 and the top address _t_&#8804;2^MXLEN^,
```

Comment:
I think `&#8804;` is `<=`. Maybe `&geq;` works and is more readable?

Reply:
Sorry, I tried but asciidoc complained, so I left it as is. Its very minor, so I will resolve this comment.

Reply:
But I mean the symbol is wrong isn't it? Don't you mean `>=` not `<=`?

Reply:
Ah right! Sorry, I did not realize you meant that the sign needs to change. Fixed! However, the `&ge;` code didn't work.